Lib/test/test_tempfile.py

```diff
@@ -886,6 +886,22 @@ def make_file():
             self.assertEqual(l, lines[i])
         self.assertEqual(i, len(lines) - 1)
 
+    def test_next(self):
+        lines = [b'spam\n', b'eggs\n', b'beans\n']
+        def make_file():
+            f = tempfile.NamedTemporaryFile(mode='w+b')
+            f.write(b''.join(lines))
+            f.seek(0)
+            return f
+
+        with make_file() as fd:
```

Comment:
This `make_file()` function seems quite unnecessary.

Reply:
this is basically a direct copy of `test_iter()` on line 876. But I can remove the `make_file()` from this case.

Reply:
removed